contrib/cirrus/runner.sh

```diff
@@ -51,11 +35,14 @@ function _run_automation() {
 }
 
 function _run_validate() {
-    # Confirm compile via prior task + cache
-    bin/podman --version
-    bin/podman-remote --version
+    # git-validation tool fails if $EPOCH_TEST_COMMIT is empty
+    # shellcheck disable=SC2154
+    if [[ -n "$EPOCH_TEST_COMMIT" ]]; then
+        make validate
```

Comment:
Confirmed that [this step executed correctly](https://cirrus-ci.com/task/6100705229406208?command=main#L0)

Reply:
oh...that reminds me.  We can override any of the `$CIRRUS_xyz` variables under the global `env:` section in `.cirrus.yml`.  That means it's possible to "fake" a branch or tag push in a PR (temporarily).  I'll do that next, so we have a proper test of all code-paths...